runtime/src/main/java/org/corfudb/protocols/wireprotocol/NettyCorfuMessageEncoder.java

```diff
@@ -12,12 +12,26 @@
 @Slf4j
 public class NettyCorfuMessageEncoder extends MessageToByteEncoder<CorfuMsg> {
 
+
+    Integer max = 0;
```

Comment:
This doesn't seem to be safe for concurrent execution. Try using LongAccumulator.